### ISSUE TITLE
Aosp 10 v203 Mi 9se FOD

### ISSUE THREAD

**Issue by Mhystes** [phhusson/treble_experimentations#859]
Hello! Everything works exept FOD on mi 9se 

(Global vendor 11.0.4 and previous).

Can't add fingerprint so

**Comment by phhusson** [comment]
Fix already committed, will be available in v204

**Comment by phhusson** [comment]
Shit wrong issue sorry

**Comment by phhusson** [comment]
Can you be more specific about your issue?
Do you see the green dot?

**Comment by Mhystes** [comment]
Yes the dot appear and i can press it but my fingerprint is not saved and the sensor doesn't vibrate.

**Comment by phhusson** [comment]
K that sucks, that means Xiaomi broke stuff compared to Mi 9.

Please link to a dump of stock rom, and please dump the result of the command `adb shell lshal`

**Comment by Mhystes** [comment]
Here the result of the commad : 

https://pastebin.com/V9LgjY9P

**Comment by XDavide** [comment]
It is fixed on the latest 206?

**Comment by Mhystes** [comment]
Nope same problem

**Comment by phhusson** [comment]
Ok, I checked your MIUI's Cit code, everything looks identical to Mi9.

Please provide logcat at boot, and when trying to enroll your fingerprint.

**Comment by Mhystes** [comment]
Here it is ! The first is at boot and the second when trying to enroll a FP. Hope you have what you need.
[logcat boot.txt](https://github.com/phhusson/treble_experimentations/files/3939524/logcat.boot.txt)
[logcat try enrollement FP.txt](https://github.com/phhusson/treble_experimentations/files/3939525/logcat.try.enrollement.FP.txt)

**Comment by dbjungle** [comment]
I'm having the same issue as well.

**Comment by dbjungle** [comment]
I flashed V10.3.3.0.PFBMIXM (Android 9) global and re-flashed PE GSI and it works now.  Previously I started from V11.0.6.0.PFBMIXM (Android 10).
